css/selectors/focus-visible-007.html

```diff
@@ -10,98 +10,48 @@
   <script src="/resources/testdriver.js"></script>
   <script src="/resources/testdriver-vendor.js"></script>
   <style>
-    [data-hadkeydown] :focus-visible {
-      outline: darkgreen solid 5px;
+    @supports not (selector(:focus-visible)) {
```

Comment:
I think you can remove the `@supports` and just use

```css
      :focus {
        outline: red solid 5px;
      }
```
since, if `:focus-visible` is supported, the `:focus-visible` and `:focus:not(:focus-visible)` below will have more precedence.

Reply:
Thanks for the review.

True thing, I could remove that. But in that case I'd probably like to add a comment about "This is to detect a problem in the tests for browsers that don't support :focus-visible". Which is already quite clear with the `@supports`.

Also most tests on this suite use that, maybe we can do a follow-up patch to change that in more places if you think it's really important.

Reply:
Fine, not really important